static/gsApp/views/amCheckout/index.tsx

```diff
@@ -863,11 +893,42 @@ class AMCheckout extends Component<Props, State> {
             )}
           </SidePanel>
         </CheckoutContainer>
-      </Fragment>
+      </ParentComponent>
     );
   }
 }
 
+const FullScreenContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: ${p => p.theme.background};
+`;
+
+const FullScreenHeader = styled('header')`
+  width: 100%;
+  border-bottom: 1px solid ${p => p.theme.border};
+  display: flex;
+  justify-content: center;
+`;
+
+const HeaderContent = styled('div')`
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  padding: ${p => p.theme.space['2xl']};
+  max-width: ${p => p.theme.breakpoints.xl};
+`;
+
+const BackButton = styled(Button)`
+  align-self: flex-start;
+  padding: 0;
+
+  & span {
+    margin-left: ${p => p.theme.space.sm};
```

Comment:
Hmm, maybe it's better to give the entire button a margin offset. cc @JonasBa curious what you think here.

I assume you're doing this to align the arrow to the left side of the content right?

Reply:
yup!